lib/types.ts

```diff
@@ -7,6 +7,7 @@ export interface Webhook<TExample extends object = object> {
   name: string;
   description: string;
   actions: string[];
+  properties: { [key: string]: { type: string; description: string } };
```

Comment:
```suggestion
  properties: Record<string, { type: string; description: string }>;
```

nit: you can use `Record for this